tg_api/silkworm_tg_api.h

```diff
@@ -40,7 +40,7 @@
 extern "C" {
 #endif
 
-enum SilkwormStatusCode {
+enum class SilkwormStatusCode {
```

Comment:
This shouldn't be `enum class` because it's C, not C++.

Reply:
Yes right.